packages/twenty-server/src/engine/core-modules/twenty-config/utils/convert-config-var-to-app-type.util.ts

```diff
@@ -0,0 +1,45 @@
+import { ConfigVariables } from 'src/engine/core-modules/twenty-config/config-variables';
+
+/**
+ * Convert a database value to the appropriate type for the environment variable
+ */
+export const convertConfigVarToAppType = <T extends keyof ConfigVariables>(
+  dbValue: unknown,
+  key: T,
+): ConfigVariables[T] => {
+  if (dbValue === null || dbValue === undefined) {
+    return dbValue as ConfigVariables[T];
+  }
+
+  // Get the default value to determine the expected type
+  const defaultValue = new ConfigVariables()[key];
```

Comment:
every time we call the util, this will instanciate a new class, is this what we want? (probably a memory leak?)